astroquery/utils/decorators.py

```diff
@@ -1,180 +1,20 @@
+from astropy.utils.decorators import deprecated as _deprecated
+from astropy.utils.decorators import (deprecated_renamed_argument
```

Comment:
no need for this, the module is entirely for internal usage, remove it altogether rather than deprecate.